test/e2e/apimachinery/aggregator.go

```diff
@@ -56,7 +56,15 @@ const (
 var _ = SIGDescribe("Aggregator", func() {
 	var ns string
 	var c clientset.Interface
-	var aggrclient *aggregatorclient.Clientset
+
+	config, err := framework.LoadConfig()
+	if err != nil {
+		framework.Failf("could not load config: %v", err)
+	}
+	aggrclient, err := aggregatorclient.NewForConfig(config)
```

Comment:
I still think this is weird, but if it's isolated to just this area I'm ok with it.